content/docs/gke/gcp-e2e.md

```diff
@@ -803,31 +707,23 @@ Follow these steps to build an image from your code:
 
 Follow these steps to deploy the web UI to your Kubeflow cluster:
 
-1. Move back into your ksonnet application directory:
+The example comes with a simple web front end that can be used with your model.
 
-    ```
-    cd ${WORKING_DIR}/my_ksonnet_app
-    ```
+Enter the `front` directory:
 
-1. Set the ksonnet parameters, using service type 
-   [LoadBalancer](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) 
-   so that you can connect with your Flask server from outside the cluster:
+```
+cd front
+```
 
-    ```
-    ks param set web-ui image ${UI_IMG_PATH}
-    ks param set web-ui type LoadBalancer
-    ```
+Optionally update the image to the ${UI_IMG_PATH} in the `deployment.yaml`:
 
-1. [Apply][ks-apply] the component to your cluster:
+To deploy the web front end
 
-    ```
-    ks apply default -c web-ui
-    ```
+```
+kustomize build . |kubectl apply -f -
```

Comment:
The pipe character looks odd here. Is it in the right place?

Reply:
That's correct, thanks.